data/react-entrypoint-generator.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * This is a script that generates static React class files for two 
+ * different scenarios:
+ *
+ * 	1. A state page that displays all the congressmen in that
+ * 	   state.
+ * 	2. A district page that displays all the congressmen in that
+ * 		 district (typically based on full address).
+ */
+const React = require('react');
+const fs = require('fs');
+const forEach = require('lodash/forEach');
+const forOwn = require('lodash/forOwn');
+const syncExec = require('sync-exec');
+
+const govData = JSON.parse(
+	fs.readFileSync('./data/people.json', 'utf8')
+);
+
+// Loop through the government data and start to sort them by
+// state and district.
+let congressByStateAndDistrict = {};
+
+forEach(govData, function(person) {
+	if (!person.current) {
+		console.log(`${person.person.name} is not a current worker in government.`);
+		return;
+	}
+
+	// If this isn't a congressman with a district, it is likely a Senator.
+	// We don't want to deal with Senator data just yet.
+	if (!person.district) {
+		console.log(`${person.person.name} is not a Congressman.`);
+		return;
+	}
+
+	let district = parseInt(person.district, 10);
+	if (isNaN(district)) {
+		console.log(`District ${district} is not numeric`);
+		return;
+	}
+
+	let state = person.state;
+
+	// Add this person object to the associated location in the
+	// congressByStateAndDistrict object.
+	if (!congressByStateAndDistrict[state]) {
+		congressByStateAndDistrict[state] = [];
+	}
+
+	congressByStateAndDistrict[state][district] = person;
+}); 
+
+console.log('Finished breaking down congressmen by district and state.');
+console.log('Now attempting to generate a Root react component file dynamically based on states and districts.');
+
+// Create a react root component file that can be made into HTML with
+// contextual data.
+const classString = 
+`var React = require('react');
+
+class Root extends React.Component {
```

Comment:
Hmm. Are you sure we won't run into browser compatibility issues on older browsers if we don't include the head and body tags? Agree with the separate file, but hesitant to remove some of the tags.